app/gui2/src/components/GraphEditor/GraphNode.vue

```diff
@@ -107,6 +107,15 @@ const error = computed(() => {
   }
 })
 
+const warning = computed(() => {
+  const externalId = graph.db.idToExternal(nodeId.value)
+  if (!externalId) return
+  const info = projectStore.computedValueRegistry.db.get(externalId)
+  const warning = info?.payload.type === 'Value' ? info.payload.warnings?.value : undefined
+  if (!warning) return undefined
```

Comment:
I think `return` or `returned undefined` are both fine but let's not mix styles within one function